test/fixtures.yml

```diff
@@ -246,6 +246,14 @@ cyber_source:
   login: X
   password: Y
 
+cyber_source_latam_co:
```

Comment:
It looks like we don't actually use this fixture anywhere; do we need to keep it?